source/postprocess/geoid.cc

```diff
@@ -0,0 +1,975 @@
+/*
+  Copyright (C) 2011 - 2015 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+#include <deal.II/base/quadrature_lib.h>
+#include <deal.II/fe/fe_values.h>
+
+#include <aspect/utilities.h>
+#include <aspect/postprocess/geoid.h>
+#include <aspect/geometry_model/spherical_shell.h>
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+    namespace internal
+    {
+      /* Definitions for HarmonicCoefficients */
```

Comment:
All of the comments above functions in this file should go into the header file.